### PR TITLE
Dev -- Connection DB

### PR DESCRIPTION
**Pull request by MartinCayuelas**

### CODE REVIEW

File: src/controllers/contactController.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const ContactModel = require('../models/contact');
+
+export const displayContacts = async (req: any, res: any) => {
```
Comment (ThomasF34) [inline src/controllers/contactController.ts]: Don't forget to use types ... This is why we use TypeScript 😆 https://github.com/ThomasF34/api-admitech/blob/init/spec/src/routes/offre.ts

File: src/controllers/contactController.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  ContactModel.findAll({
+    order: [['name', 'ASC']]
+  }).then((contactElems: any) => {
```
Comment (ThomasF34) [inline src/controllers/contactController.ts]: Same here and on every `any`

File: src/controllers/contactController.ts (hunk 1 above)
Comment (MartinCayuelas) [reply src/controllers/contactController.ts]: Got it!

File: src/controllers/contactController.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
+const ContactModel = require('../models/contact');
```
Comment (ThomasF34) [inline src/controllers/contactController.ts]: Also, TS uses another kind of import. You should use it too. (See on the link already given 😄)

**Comment by MartinCayuelas** [comment]
ES6 on other branch
Clothing this PR

**Pull request closed without merging**